apps/cli/commands/ingest_elasticsearch.py

```
@@ -0,0 +1,81 @@
+    """
+    try:
+        config = get_config()
+        limit_str = f"limit: {limit}" if limit else "no limit"
+        console.print(f"[yellow]Starting Elasticsearch ingestion: {index} ({limit_str})[/yellow]")
+
+        # Parse query JSON
+        try:
+            query_dict = json.loads(query)
+        except json.JSONDecodeError as e:
+            console.print(f"[red]✗ Invalid JSON query: {e}[/red]")
+            raise typer.Exit(code=1)
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)